templates/components/common/navigation.html

```diff
@@ -43,79 +34,125 @@
         {{/if}}
         <li>
             <button 
-                class="js-nav-user-action js-quick-search-expand"
+                class="js-nav-user-action js-quick-search-expand c-top-nav-search c-nav-page-action"
```

Comment:
Should be rather `c-top-nav__search`, and again, mixing different component classnames on the same element